lib/Core/Executor.cpp

```diff
@@ -3012,8 +3012,8 @@ void Executor::callExternalFunction(ExecutionState &state,
     return;
   
   if (NoExternals && !okExternals.count(function->getName())) {
-    llvm::errs() << "KLEE:ERROR: Calling not-OK external function : "
-                 << function->getName().str() << "\n";
+    klee_error("Calling not-OK external function : %s\n",
```

Comment:
here the semantic changes, `klee_error` terminates klee, but it should continue.
Still, a KLEE:ERROR should be printed.
I would go `klee_warning` in this case.